Cargo.toml

```diff
@@ -14,15 +14,15 @@ members = [
 anyhow = "1"
 async-trait = "0.1"
 apache-avro = { version = "0.15", features = ["derive"] }
-arrow = { version = ">=40, <47" }
+arrow = { version = "46" }
```

Comment:
We must use `46` here? This could affect our users with other versions.

Reply:
Yes, only `46` contains this fix. BTW, is it really a good choice to use range for arrow? IIUC, arrow doesn't guarantee portability cross major versions.

Reply:
The correctness of the arrow API is guaranteed by arrow maintainer and endusers, and Icelake is just to ensure that it can compile in this case. Maybe we can use `>=46` instead?

Reply:
Agree, I will fix it.